components/progresscircle/package.json

```diff
@@ -16,15 +16,15 @@
     "build": "gulp"
   },
   "peerDependencies": {
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.1"
```

Comment:
What was vars again? Does tokens replace it?

Reply:
Looks like I had a couple issues getting this file updated. I've corrected it and pushed a [fix commit](https://github.com/adobe/spectrum-css/commit/54c65d61df7931a47fe04bf796a5640ab66ff53b)